metals/src/main/scala/scala/meta/internal/metals/testProvider/TestSuitesProvider.scala

```diff
@@ -46,18 +54,21 @@ final class TestSuitesProvider(
     userConfig: () => UserConfiguration,
     client: MetalsLanguageClient,
 )(implicit ec: ExecutionContext)
-    extends SemanticdbFeatureProvider {
+    extends SemanticdbFeatureProvider
+    with CodeLens {
 
   private val index = new TestSuitesIndex
   private val junitTestFinder = new JunitTestFinder
   private val munitTestFinder =
     new MunitTestFinder(trees, symbolIndex, semanticdbs)
   private val scalatestTestFinder =
     new ScalatestTestFinder(trees, symbolIndex, semanticdbs)
+  private val isExplorerEnabled = clientConfig.isTestExplorerProvider() &&
```

Comment:
```suggestion
  private def isExplorerEnabled = clientConfig.isTestExplorerProvider() &&
```
Let's change this as @kpodsiad suggested and we should be good to merge!

Reply:
Done!